tests/flow/test_list.py

```diff
@@ -143,3 +143,633 @@ def test03_null_list_function_inputs(self):
+        query = """MATCH (a:X) DELETE a"""
+        redis_graph.query(query)
+
+        # # List of edge input should return list of NULL
+        query = """CREATE (a:X), (b:Y)"""
+        redis_graph.query(query)
+        query = """MATCH (a:X), (b:Y) CREATE (a)-[e:R]->(b)"""
+        redis_graph.query(query)
+        query = """MATCH ()-[e]->() RETURN toIntegerList([e])"""
+        expected_result = [[None]]
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result)
+        query = """MATCH (a:X),(b:Y) DELETE a, b"""
+        redis_graph.query(query)
+
+        # List of path input should return list of NULL
+        query = """CREATE (a:X), (b:Y)"""
+        redis_graph.query(query)
+        query = """MATCH (a:X), (b:Y) create (a)-[e:R]->(b)"""
+        redis_graph.query(query)
+        query = """MATCH p=()-[]->() RETURN toIntegerList([p])"""
+        expected_result = [[None]]
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result)
+        query = """MATCH (a:X), (b:Y) DELETE a, b"""
+        redis_graph.query(query)
+
+        # Test without input argument
+        # try:
+        #     query = """RETURN toIntegerList()"""
+        #     redis_graph.query(query)
+        #     self.env.assertTrue(False)
+        # except ResponseError as e:
+        #     self.env.assertContains("Received 0 arguments to function 'toIntegerList', expected at least 1", str(e))
+
+    def test07_toStringList(self):
+        # NULL input should return NULL
+        expected_result = [None]
+        query = """RETURN toStringList(null)"""
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result)
+
+        # List of NULL values input should return list of NULL values
+        query = """RETURN toStringList([null, null])"""
+        expected_result = [[None, None]]
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result) 
+
+        # Test list with string values
+        query = """RETURN toStringList(['abc', '5.32', 'true', 'this is a test'])"""
+        expected_result = [['abc', '5.32', 'true', 'this is a test']]
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result)
+
+        # Test list with float values
+        query = """RETURN toStringList([0.32425, 5.32, 7.1])"""
+        expected_result = [['0.324250', '5.320000', '7.100000']]
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result)
+
+        # Test list with mixed type values
+        query = """RETURN toStringList(['abc', 7, '5.32', null, ['a','b']]) """
+        expected_result = [['abc', '7', '5.32', None, None]]
```

Comment:
Aren't we expecting to see a string representation of `['a','b']` ?

Reply:
I don't think so. Because toStringOrNull( list ) returns null.
```
127.0.0.1:6379> GRAPH.QUERY g "RETURN toStringOrNull(['a','b'])"
1) 1) "toStringOrNull(['a','b'])"
2) 1) 1) (nil)
```